test/extensions/EXT/SPV_EXT_relaxed_printf_string_address_space/non-constant-printf.ll

```diff
@@ -38,14 +49,16 @@
 target datalayout = "e-p:32:32:32-i1:8:8-i8:8:8-i16:16:16-i32:32:32-i64:64:64-f32:32:32-f64:64:64-v16:16:16-v24:32:32-v32:32:32-v48:64:64-v64:64:64-v96:128:128-v128:128:128-v192:256:256-v256:256:256-v512:512:512-v1024:1024:1024"
 target triple = "spir-unknown-unknown"
 
-@0 = internal unnamed_addr addrspace(0) constant [6 x i8] c"Test\0A\00", align 1
+@0 = internal unnamed_addr addrspace(2) constant [6 x i8] c"Test\0A\00", align 1
```

Comment:
Why is this change required?

Thanks

Reply:
It's exactly the reason for creation of the whole PR. addrspace(0) is a default address space (Private) that is mapped to  Function storage class in SPIR-V, violating validation rules. In this case we need to create a global variable in other than 0 address space.